Core/Inc/pdu.h

```diff
@@ -3,14 +3,15 @@
 
 #include "cmsis_os.h"
 #include "pi4ioe.h"
+#include "max7314.h"
 #include <stdbool.h>
 #include <stdint.h>
 
 typedef struct {
 	I2C_HandleTypeDef* hi2c;
 	osMutexId_t* mutex;
 	pi4ioe_t* shutdown_expander;
```

Comment:
Is this summosed to be a max7314_t ?? I do not see a pi4ioe on the board, and the shutdown controller is currently a MAX7314. If so, I will make all the necessary changes to the shutdown loop code as well.

Reply:
Yeah it's supposed to be max, was originally piowhatever but we changed it for some reason that I forget